docs/maintenance/Pull_Requests.mdx

```diff
@@ -24,15 +24,105 @@ Per the scales from [Contribution Tiers > Pointing](./Contributor_Tiers.mdx#poin
+- Otherwise, add the `please fill out the template` label and post a comment like the _Needs Issue_ template
+
+Upon completing your review, if the build is passing and you feel comfortable merging it in, go ahead and squash merge.
+Otherwise, add the `1 approval` label.
+
+#### Common Things to Look For
+
+- Style: that can you read through the code easily, there are comments for necessary tricky areas, and not unnecessary comments otherwise.
+  - Try not to nitpick things that don't matter.
+  - If something is standard in the repo but not enforced, consider mentioning it as a non-blocking comment and filing an issue to enforce it.
+- Thoroughness: does it handle relevant edge cases? Commonly:
+  - Generics and type parameters (see: `getConstrainedTypeAtLocation`).
+  - Parenthesis and whitespace (see: `getWrappingFixer`).
+  - Unions and intersections (see: `unionTypeParts` and `intersectionTypeParts`).
+- Unit tests:
+  - All lines are covered per the Codecov / `yarn jest path/to/impacted/file --coverage` report.
+  - Both "positive" and "negative" ("valid" and "invalid") cases exist, if reasonably possible to test for.
+  - Fixes and suggestions, if present, don't remove `//` or `/*` comments
+  - `invalid` lint rule errors include line and column information
+  - Valid syntax edge cases don't cause the rule to crash, even if they cause a type error in TypeScript
+
+#### Individual Comments
+
+Post about one comment per area of praise note, suggested change, or non-actionable note.
+It's fine to use multiple ancillary comments to indicate _"(also here)"_ notes, but don't overwhelm with requests.
+
+:::tip
+If you're posting more than a couple types of comments, consider prefixing them with category indicators such as `[Docs]`, `[Praise]`, `[Testing]`, etc.
+This helps avoid the review feeling like a huge slog of big requests.
+:::
+
+Be clear in each of your comments what you're looking for or saying.
+Err on the side of providing more information than you think is needed.
+
+Try to default to a questioning tone for points that aren't clear bugs.
+Encourage authors to think on your suggestions: _"I think {xyz}, but am not sure - what do you think?"_.
+
+#### Preliminary or Repeat Reviews
+
+Sometimes you may want to post a preliminary review, and/or realize later on you missed something in an earlier review.
+Both are reasonable and fine.
+
+For preliminary reviews, be clear with what scale you're reviewing at: _"Reviewing on architecture now, will look in more detail once it's settled"_.
+
+For repeat reviews, be clear when it's something you missed earlier and/or there's new information.
+Don't apologize if the missed information was only made clear because of requested changes - this is part of the review process!
+
+### Other States
+
+#### External Blockers
+
+If the PR is blocked on something, such as an external API or discussion in an issue, add the `status: blocked` and appropriate `blocked by *` label.
```

Comment:
```suggestion
If the PR is blocked on something, such as an external API or discussion in an issue, add the and appropriate `blocked by *` label.
```

Ah, there doesn't seem to be a `status: blocked` label here?

![There is no 'status: blocked' label in ts-eslint repo](https://github.com/typescript-eslint/typescript-eslint/assets/61150013/c150b0a0-cbd7-42bc-aa26-ace8b9660fba)

Reply:
Ha, I forget which repos I'm in sometimes...